### PR TITLE
Refresh the page after users Sign In / Sign Out

### PR DESCRIPTION
**Pull request by alkim91**
Refresh the page once users sign-in in Venia, and redirect users back to the main storefront once they sign out.

**DO NOT MERGE before https://github.com/adobe/aem-core-cif-components/pull/442 has been merged** 

## Description

Sign-Out : 
I added the baseUrl(Storefront URL) in the App configs, and in the event handler for signing out, we redirect the user to the main storefront once they click 'Sign Out'. The baseUrl is configured based on the main storefront url(Storefront can be any language/country, not always US and English), and is implemented in StoreConfigExporter(See https://github.com/adobe/aem-core-cif-components/pull/442)

Sign-In : 
Simple refresh.

## Motivation and Context

If users are in any admin pages(Address book, account management), once they sign out we want them to be redirected back to the main storefront. Currently, we revoke the user token and reset cookies as well as the contexts, but no refresh/redirect takes place.

## How Has This Been Tested?

This has been tested in the two sign out user cases.

1. Signing out from the main navigation. Once we click 'sign out', no matter which page we're in(Catalog, address book, etc), we're taken back to the storefront.

2. Same thing happens when signing out from the account panel in responsive mode.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [x] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.frontend/src/main/components/App/App.js [hunk 1]
```diff
@@ -35,11 +35,10 @@ import partialConfig from './config';
 import '../../site/main.scss';
 
 const App = () => {
-    const { storeView, graphqlEndpoint, graphqlMethod } = document.querySelector('body').dataset;
```
Comment (herzog31) [inline ui.frontend/src/main/components/App/App.js:38]: Can you please add `storeView` to the config again? That's an important property used by the components and should not be removed.

File: ui.frontend/src/main/components/App/App.js [hunk 2]
```diff
@@ -35,11 +35,10 @@ import partialConfig from './config';
...
-    const { mountingPoints, pagePaths } = partialConfig;
+    const { graphqlEndpoint, graphqlMethod } = document.querySelector('body').dataset;
+    const { mountingPoints, pagePaths, baseUrl } = partialConfig;
```
Comment (herzog31) [inline ui.frontend/src/main/components/App/App.js:39]: The `baseUrl` variable is unused here.

File: ui.frontend/src/main/components/App/App.js (hunk 1 above)
Comment (alkim91) [reply ui.frontend/src/main/components/App/App.js:38]: Ah you're right. I just put it back.

File: ui.frontend/src/main/components/App/App.js (hunk 2 above)
Comment (alkim91) [reply ui.frontend/src/main/components/App/App.js]: Removed.

File: ui.frontend/src/main/components/App/App.js (hunk 1 above)
Comment (herzog31) [reply ui.frontend/src/main/components/App/App.js]: Please add it back to the config as well.

Comment (alkim91) [reply ui.frontend/src/main/components/App/App.js:38]: Yep should be fixed now.

File: ui.frontend/src/main/components/App/App.js [hunk 3]
```diff
@@ -38,8 +38,8 @@ const App = () => {
     const { storeView, graphqlEndpoint, graphqlMethod } = document.querySelector('body').dataset;
     const { mountingPoints, pagePaths } = partialConfig;
     const config = {
-        ...partialConfig,
         storeView,
+        ...partialConfig,
```
Comment (herzog31) [inline ui.frontend/src/main/components/App/App.js:42]: Is there a reason to change the order of `storeView` and `...partialConfig` here? :)

Comment (alkim91) [reply ui.frontend/src/main/components/App/App.js]: Nope - I put it back just now.

**Pull request merged**